prisma/seed.ts

```diff
@@ -0,0 +1,103 @@
+import { PrismaClient } from "@prisma/client";
+const prisma = new PrismaClient();
+
```

Comment:
```suggestion
import defaults from "permissions/defaults";
// defaults.PERMISSIONS_MANAGER
// defaults.PERMISSIONS_INSPECTOR
// etc
```

Reply:
Suggestion - can resolve

Reply:
The seed file compiler didn't approve of imports, so I've just dumped the defaults into the seed file itself for the moment.